mydemoproject/demoapp1/models/book.py

```diff
@@ -46,5 +40,4 @@ def add_book(cls, b_name, aut_name, **extra_fields):
             return True, book
 
         else:
-            return False, ("Cant add book because it already exists "
-                           "for that particular author name")
+            return False, "Can't add already existing book."
```

Comment:
The below message conveys the error and is a correct sentence, not a half phrase.
```suggestion
            return False, "Book already exists."
```

Reply:
done